comit/src/expiries/config.rs

```diff
@@ -34,16 +34,25 @@ use time::Duration;
 // to turn off lints.
 
 mod main {
-    pub const BITCOIN_BLOCK_TIME_SECS: u16 = 600; // 10 minutes, average Bitcoin block time.
-    pub const ETHEREUM_BLOCK_TIME_SECS: u16 = 20; // Conservative Ethereum block
-                                                  // time.
-
-    pub const BITCOIN_CONFIRMATIONS: u8 = 6; // Standard in the Bitcoin ecosystem.
-    pub const ETHEREUM_CONFIRMATIONS: u8 = 30; // Value used by Kraken.
-
-    pub const ACT_IN_SOFTWARE_SECS: u32 = 15 * 60; // Value arbitrarily chosen.
-    pub const ACT_WITH_USER_INTERACTION_SECS: u32 = 60 * 60; // Value arbitrarily
-                                                             // chosen.
+    use super::*;
+    use conquer_once::Lazy;
+    use time::NumericalDuration;
+
+    // 10 minutes, average Bitcoin block time.
+    pub static BITCOIN_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| 600.seconds());
+    // Conservative Ethereum block // time.
+    pub static ETHEREUM_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| 20.seconds());
+
+    // Standard in the Bitcoin ecosystem.
+    pub const BITCOIN_CONFIRMATIONS: u8 = 6;
+    // Value used by Kraken.
+    pub const ETHEREUM_CONFIRMATIONS: u8 = 30;
+
+    // Allows someone running this as a software to react to any failure within 24
+    // hours.
+    pub static ACT_IN_SOFTWARE: Lazy<Duration> = Lazy::new(|| 24.hours());
```

Comment:
The software can fail and the operator may not be in front of a computer for 24 hours.
The user is expected to be in front of the computer.

The meaning hasn't changed, the only difference is now we are matching the concept to real life use.

Reply:
Fair enough. This time period is added for every time there is an action in software - that's why the expiries are so long once this is applied. I'm not acking or nacking just making you aware of the ramifications in case you missed them :)